site/src/components/LicenseCard/LicenseCard.tsx

```diff
@@ -30,6 +30,9 @@ export const LicenseCard = ({
     number | undefined
   >(undefined)
 
+  const currentUserLimit =
+    license.claims.features["user_limit"] || userLimitLimit
```

Comment:
This is probably out of scope of this PR, but `userLimitLimit` sounds weird.

Reply:
I find it weird as well